src/controllers/reset/index.ts

```diff
@@ -0,0 +1,115 @@
+import { Get, Hidden, Produces, Route, SuccessResponse } from 'tsoa'
+import { inject, injectable, singleton } from 'tsyringe'
+
+import { Env } from '../../env.js'
+import { BadRequestError, InternalError } from '../../errors.js'
+import { Logger, type ILogger } from '../../logger.js'
+import Database from '../../models/db/index.js'
+import type { ConnectionRow } from '../../models/db/types.js'
+import VeritableCloudagent from '../../models/veritableCloudagent.js'
+
+@singleton()
+@injectable()
+// @Security('oauth2')
+@Route('/reset')
+@Hidden()
+@Produces('text/html')
+export class ResetController {
+  constructor(
+    private env: Env,
+    private db: Database,
+    private cloudagent: VeritableCloudagent,
+    @inject(Logger) private logger: ILogger
+  ) {
+    this.logger = logger.child({ controller: '/reset' })
+  }
+
+  // would be nice to restore to prior reset
+  public async isReset(): Promise<boolean> {
```

Comment:
is there a need for this to be public?

Reply:
mm... pretty sure it was due to sinon.stub. yea, otherwise it would create quite a few mocks in terms of db checks like onFisrtCall and etc.... managed to stub a private method so should be reflected in the next push